spec/cohortManager.spec.js

```diff
@@ -0,0 +1,218 @@
+const { CohortManager, Cohort, Student } = require('./../src/cohortManager.js')
+
+describe('cohortManager', () => {
+  it('should pass', () => {
+    expect(true).toEqual(true)
+  })
+})
+
+describe('Cohort Manager', () => {
+  let cohortManager
+
+  beforeEach(() => {
+    cohortManager = new CohortManager()
+  })
+
+  it('Should create a cohort with a name', () => {
+    // setup
+    const cohort1 = new Cohort('Cohort 1')
+    const expected = [cohort1]
+    // execute
+    const result = cohortManager.createCohort('Cohort 1')
+    // verify
+    expect(result).toEqual(expected)
+  })
+
+  it('Should create a student with a unique ID', () => {
+    // setup
+    const student1 = new Student('John', 'Deere')
+    student1.studentId = 1
```

Comment:
property for student could be named `id` so you write code like this `student1.id`

Reply:
done!